split_tunnel.go

```diff
@@ -0,0 +1,105 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// SplitTunnelIncludeGetResponse represents the response from the get split
+// tunnel endpoints.
+type SplitTunnelGetResponse struct {
+	Response
+	Result []SplitTunnel `json:"result"`
+}
+
+// SplitTunnel represents the individual tunnel struct.
+type SplitTunnel struct {
+	Address     string `json:"address,omitempty"`
+	Host        string `json:"host,omitempty"`
+	Description string `json:"description,omitempty"`
+}
+
+
+// SplitTunnelInclude returns all split tunnel include rules within an account.
+//
+// API reference: https://api.cloudflare.com/#device-policy-get-split-tunnel-include-list
+func (api *API) SplitTunnelInclude(ctx context.Context, accountID string) ([]SplitTunnel, error) {
```

Comment:
I was having a look at the Terraform implementation and wondered if we would be better off passing in the `include`/`exclude` into this method as a parameter and constructing the URL using the value (similar to what we do with the other parameters already)?

We should also prefix this method with `List` seeing how we are pulling them all back and it will line up with #619 when we get to it.

Reply:
That makes sense. I was originally thinking that, but then thought if they ever diverged that it would be better to have two separate implementations, but we can cross that bridge if we get to it. I also combined the update functions since those are the exact same as well and can just be constructed using an `include/exclude` parameter. Let me know if that was a bad idea.